packages.json

```diff
@@ -6453,5 +6453,16 @@
     "description": "Shiny bindings to the JACK Audio Connection Kit.",
     "license": "MIT",
     "web": "https://github.com/Skrylar/nim-jack"
+  },
+  {
+    "name": "serializetools",
+    "url": "https://github.com/JeffersonLab/serializetools",
+    "method": "git",
+    "tags": [
+      "yes"
```

Comment:
Some tags would be nice here :)

I can add them for you if you don't wish to mess with the PR. Just tell me which to add.